AddonExample/AddonExample.cs

```diff
@@ -1,57 +1,56 @@
-﻿using RainbowMage.OverlayPlugin;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using Advanced_Combat_Tracker;
+using RainbowMage.OverlayPlugin;
+using System.Windows.Forms;
 
 namespace AddonExample
 {
-    public class AddonExample : IOverlayAddon
+    public class AddonExample : IActPluginV1, IOverlayAddonV2
     {
-        public string Name
-        {
-            get { return "Addon Example"; }
-        }
+        public static string pluginPath = "";
 
-        public string Description
+        public void DeInitPlugin()
         {
-            get { return "Just displays www.yahoo.com."; }
-        }
 
-        public Type OverlayType
-        {
-            get { return typeof(AddonExampleOverlay); }
         }
 
-        public Type OverlayConfigType
+        public void InitPlugin(TabPage pluginScreenSpace, Label pluginStatusText)
         {
-            get { return typeof(AddonExampleOverlayConfig); }
-        }
+            pluginStatusText.Text = "Ready.";
 
-        public Type OverlayConfigControlType
-        {
-            get { return typeof(AddonExampleOverlayConfigPanel); }
-        }
+            // We don't need a tab here.
+            ((TabControl)pluginScreenSpace.Parent).TabPages.Remove(pluginScreenSpace);
 
-        public IOverlay CreateOverlayInstance(IOverlayConfig config)
-        {
-            return new AddonExampleOverlay((AddonExampleOverlayConfig)config);
+            foreach (var plugin in ActGlobals.oFormActMain.ActPlugins)
+            {
+                if (plugin.pluginObj == this)
+                {
+                    pluginPath = plugin.pluginFile.FullName;
+                    break;
+                }
+            }
         }
 
-        public IOverlayConfig CreateOverlayConfigInstance(string name)
+
+        public string Name
         {
-            return new AddonExampleOverlayConfig(name);
+            get { return "Addon Example"; }
         }
 
-        public System.Windows.Forms.Control CreateOverlayConfigControlInstance(IOverlay overlay)
+        public string Description
         {
-            return new AddonExampleOverlayConfigPanel();
+            get { return "Addon Example Description"; }
```

Comment:
You can remove the `Name` and `Description` properties. OverlayPlugin doesn't use them.

Reply:
Delete them by d7924cc